docs/_reference/oauth.md

```diff
@@ -148,17 +122,34 @@ slug: oauth
 </thead>
 <tbody>
 <tr>
+<td align="center">req</td>
+<td align="center"><a href="https://nodejs.org/api/http.html#class-httpincomingmessage"><code>IncomingMessage</code></a></td>
+<td align="center">✓</td>
+<td></td>
+</tr>
+<tr>
+<td align="center">res</td>
+<td align="center"><a href="https://nodejs.org/api/http.html#class-httpserverresponse"><code>ServerResponse</code></a></td>
+<td align="center">✓</td>
+<td></td>
+</tr>
+<tr>
 <td align="center">options</td>
+<td align="center"><code><a href="#callbackoptions" title="">CallbackOptions</a></code></td>
+<td align="center">✗</td>
+<td></td>
+</tr>
+<tr>
+<td align="center">installOptions</td>
 <td align="center"><code><a href="#installurloptions" title="">InstallURLOptions</a></code></td>
-<td align="center">✓</td>
+<td align="center">✗</td>
 <td></td>
 </tr>
 </tbody>
 </table>
-<p><strong>Returns</strong> <code>Promise&#x3C;string></code></p>
-<h4>handleCallback(req, res, options)</h4>
-<p>This method handles the incoming request to the callback URL. It can be used as a RequestListener in almost any HTTP server framework.</p>
-<p>Verifies the state using the stateStore, exchanges the grant in the query params for an access token, and stores token and associated data in the installationStore.</p>
+<p><strong>Returns</strong> <code>Promise&#x3C;void></code></p>
+<h4>handleInstallPath(req, res, options, installOptions)</h4>
```

Comment:
How about moving this method to the top in this section? handleInstallPath & generateInstallUrl -> handleCallback -> authorize would be easier to understand for readers.

Reply:
Sounds good, I was mainly trying to keep the entries in alphabetic order but this makes more sense.